en/identity-server/7.1.0/docs/apis/restapis/authenticators.yaml

```diff
@@ -138,6 +138,137 @@ paths:
             curl --location 'https://localhost:9443/api/server/v1/authenticators/meta/tags' \
             -H 'Accept: application/json' \
             -H 'Authorization: Basic YWRtaW46YWRtaW4='
+  /authenticators/custom:
+    post:
+      tags:
+        - User defined local authenticators
+      summary: |
+        Create a new user defined local authenticator.
+      description: |
+        This API provides the capability to create a new user defined local authenticator. 
+        
+        <b>Scope(Permission) required:</b> `internal_custom_authenticator_create`
+      responses:
+        '201':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Authenticator'
+              examples:
+                example:
+                  $ref: '#/components/examples/UserDefinedAuthenticatorExample'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserDefinedLocalAuthenticatorCreation'
+        description: This represents the user defined local authenticator to be created.
+        required: true
+  /authenticators/custom/{authenticator-id}:
+    put:
+      tags:
+        - User defined local authenticators
+      summary: |
+        Update a user defined local authenticator.
+      description: |
+        This API provides the capability to update a user defined local authenticator configurations. <br>
+
+        <b>Scope(Permission) required:</b> `internal_custom_authenticator_update`
+      parameters:
+        - name: authenticator-id
+          in: path
+          description: ID of an authenticator
```

Comment:
Shall we say Unique identifier of an authenticator ?

Reply:
or is this added to be consistent with terms used in the api today?

Reply:
Also I think we should mention that for custom authenticators the id should always start with 'custom-' prefix

Reply:
For `/authenticators/{authenticator-id}/connected-apps` in the same yaml, uses the `ID of an authenticator` pharse.